cypress/e2e/cache-create-wizard-template.cy.js

```diff
@@ -19,11 +19,71 @@ describe('Cache Creation Wizard', () => {
     cy.get('#template-selector').click();
     cy.contains('e2e-test-template').parent().find('button').click();
     cy.get('[data-cy=wizardNextButton]').click();
+    cy.contains('Cache ' + cacheName + ' successfully created with e2e-test-template.');
     // Once the cache created, redirection to main page is done and the cache should be visible
     //Is redirected to Data Container page
     cy.get('#cluster-manager-header').should('exist');
     cy.get('[data-cy=cacheManagerStatus]').should('exist');
     cy.get('[data-cy=rebalancingSwitch]').should('exist');
     cy.contains(cacheName);
   })
+
+//   it.only('successfully creates without a template a JSON config', () => {
```

Comment:
@andyuk1986 are these tests going to be back non commented?

Reply:
@karesti yes, at the moment I have an issue with cypress typing JSON/XML to cache creation textarea. These tests were passing with Cypress 10.4.x but then after upgrade to 10.8.x and now to 11.x.x they are failing because while typing the cypress "eats" symbols. As soon as I find the solution (or maybe it is a bug on cypress side, not sure), I will uncomment the tests.